### DISCUSSION TITLE
Memory bug

### DISCUSSION THREAD

**Discussion by 1liuzhongyi** [termux/termux-app#3938] [General]
每打开一次 termux，内存就会增加 1mb
可在“设置/开发者选项/正在运行的服务”里看到内存使用情况
Memory increases by 1mb each time termux is opened
Memory usage can be seen in "Settings / Developer Options / Running Services"

**Comment by twaik** [comment]
No need to post it more than once. 
Closing this in prior to #3939.

**Reply by 1liuzhongyi** [reply]
我不太会玩 github
